tests/Extensions/TimedScopes.UnitTests/ActivityObserversIntializerTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Omex.Extensions.Abstractions.Activities.Processing;
+using Microsoft.Omex.Extensions.TimedScopes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Hosting.Services.UnitTests
+{
+	[TestClass]
+	public class ActivityObserversIntializerTests
+	{
+		[TestMethod]
+		public async Task ActivityObserversInvokedProperly()
+		{
+			string name = nameof(ActivityObserversInvokedProperly);
+			Mock<IActivityStartObserver> startObserver = new Mock<IActivityStartObserver>();
+			Mock<IActivityStopObserver> stopObserver = new Mock<IActivityStopObserver>();
+			ActivityObserversIntializer initializer = new ActivityObserversIntializer(
+				new [] { startObserver.Object },
+				new [] { stopObserver.Object });
+
+			try
+			{
+				await initializer.StartAsync(CancellationToken.None);
```

Comment:
>StartAsync [](start = 22, length = 10)

I'd suggest adding `.ConfigureAwait(false)` here to the async call here and below for consistency with the product code

Reply:
Sorry missed this one, thank you, fixed